numpy/core/src/umath/simd.inc.src

```diff
@@ -1333,6 +1367,7 @@ static NPY_GCC_OPT_3 NPY_GCC_TARGET_@ISA@ void
     @vtype@ inf = _mm@vsize@_set1_ps(NPY_INFINITYF);
     @vtype@ zeros_f = _mm@vsize@_set1_ps(0.0f);
     @vtype@ poly, num_poly, denom_poly, quadrant;
+    @vtype@i vindex = _mm@vsize@_loadu_si@vsize@((@vtype@i*)&indexarr[0]);
```

Comment:
this is only needed for stride != 1 so it should and its variable declarions be moved into that if condition for minimal scope

Reply:
nevermind, the other code is in the loop which would be slow